genesis/ext/pyrender/utils.py

```diff
@@ -87,15 +87,23 @@ def format_texture_source(texture, target_channels="RGB"):
             if texture.shape[2] == 1:
                 texture = np.repeat(texture, 3, axis=2)
             elif texture.shape[2] == 2:
-                raise ValueError("Cannot reformat 2-channel texture into RGB")
+                l = texture[:, :, 0]
+                texture = np.stack([l, l, l], axis=2)
```

Comment:
Just do `texture = texture[:, :, (0, 0, 0)]`.

Plus add a comment to explain what it does (as you did in the description of the PR).